darwin/DarwinProcess.c

```diff
@@ -269,6 +269,20 @@ static void DarwinProcess_updateCmdLine(const struct kinfo_proc* k, Process* pro
    Process_updateCmdline(proc, k->kp_proc.p_comm, 0, strlen(k->kp_proc.p_comm));
 }
 
+// Converts ticks in the Mach "timebase" to nanoseconds.
+// See `mach_timebase_info`, as used to define the `Platform_timebaseToNS` constant.
+static uint64_t machTicksToNanoseconds(uint64_t schedulerTicks) {
+   double nanoseconds_per_mach_tick = Platform_timebaseToNS;
+   return (uint64_t) (nanoseconds_per_mach_tick * (double) schedulerTicks);
+}
+
+// Converts nanoseconds to hundreths of a second (centiseconds) as needed by the "time" field of the Process struct.
+static long long int nanosecondsToCentiseconds(uint64_t nanoseconds) {
+   uint64_t centiseconds_per_second = 100;
```

Comment:
Can prefix with 'const' here I think.

Reply:
How thoroughly should I `const` things up? A whole bunch of local vars in `DarwinProcess_setFromLibprocPidinfo` could be made const, too.

Reply:
> How thoroughly should I `const` things up? A whole bunch of local vars in `DarwinProcess_setFromLibprocPidinfo` could be made const, too.

For now, just use const where it makes sense in the places you're changing/adding code.  Over time it'd be good to move other darwin/ code towards wider use of const like has been done in the htop core code.